dcp-0010/dcp-0010.mediawiki

```diff
@@ -0,0 +1,448 @@
+
+===Modified Subsidy Split===
+
+The subsidy split for the full block subsidy MUST be split between Proof-of-Work
+(PoW), Proof-of-Stake (PoS), and the decentralized Treasury as follows:
+
+* PoW - 10%
+* PoS - 80%
+* Treasury - 10% (no change from existing)
+
+Additionally, the resulting PoS subsidy MUST be split evenly between the maximum
+number of votes per block, which, as of this specification, is 5 on all
+networks.
+
+The Treasury subsidy proportion MUST remain the same and otherwise retain all
+existing semantics it has prior to this specification.
+
+The following formulas precisely specify the calculations for the modified
+subsidy splits for arbitrary heights.  This ensures the semantics are fully
+specified for all heights; however, it is important to note that they MUST
+only be applied to <code>mainnet</code>, and current <code>testnet</code>, for
+heights where the new rules are active per the associated on-chain vote.
+
+<!--S_{pow}(h,v) & = & \begin{cases}0 & \text{if } h \le 0,\\S_{full}(1) & \text{if } h = 1,\\\left\lfloor\frac{S_{full}(h) \times 1}{10}\right\rfloor & \text{if } 2 \le h < H_{sv},\\0 & \text{if } h \ge H_{sv} \textbf{ and } v < v_{m},\\\left\lfloor\frac{\left\lfloor\frac{S_{full}(h) \times 1}{10}\right\rfloor \times v}{v_{b}}\right\rfloor & \text{if } h \ge H_{sv} \textbf{ and } v \ge v_{m},\end{cases}-->
+<!--S_{vote}(h) & = & \begin{cases}0 & \text{if } 0 \le x < H_{sv}-1,\\\left\lfloor\frac{S_{full}(h) \times 8}{10 \times v_{b} }\right\rfloor & \text{if } x \ge H_{sv}-1,\end{cases}-->
+<img src="subsidy_proportions.svg" width="100%" height="464" />
+
+Explanation of terms:
+
+S<sub>full</sub> = The full block subsidy at a given height<br />
+S<sub>pow</sub> = The subsidy for proof of work at a given height for a given number of votes<br />
+S<sub>vote</sub> = The subsidy for a single vote at a given height<br />
+h = The height for which the subsidy is to be calculated<br />
+v = The number of votes included in the block
+H<sub>sv</sub> = The height at which stake validation begins (4096 on <code>mainnet</code>)<br />
+v<sub>b</sub> = The number of votes per block (5 on all networks)
+v<sub>m</sub> = The minimum number of required votes (3 on all networks)
+
+===Subsidy Enforcement===
+
+IMPORTANT: The PoS subsidy, and hence each individual  vote subsidy, is based on
+the block PRIOR to the one that contains the vote since that prior block is the
+one being voted on.  In practice, this means the formula to calculate the vote
+subsidy MUST be called with a height of one less than the PoW subsidy formula
+once the height that voting begins at is reached.
+
+The following modified rules MUST be enforced:
+
+* The input value of the one and only input of the coinbase transaction MUST commit to the modified PoW subsidy proportion
+* The total of the output values of the coinbase transaction MUST NOT exceed the modified PoW subsidy proportion plus the total transaction fees
```

Comment:
"plus the total transaction fees" could be clearer, indicating that it's talking about the fees from the other transactions in the block.  The word "total" sort of implies that, but it could be more explicit.

Reply:
Updated to `plus the total fees from all transactions contained in the block.`